src/styles/blocks/nav.scss

```
@@ -37,5 +23,10 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    transition: 0.3;
```

Необходимо явно указывать свойства, которые будут анимированы, иначе используется transition: all
transition: all достаточно требователен к ресурсам браузера и на старых компьютерах может ухудшать пользовательский опыт (check everywhere)

```suggestion
    transition: color 0.3;
```